projects/ad400x-fmcz/src/ad400x_fmcz.c

```diff
@@ -0,0 +1,113 @@
+#include <stdio.h>
+#include <sleep.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <xil_cache.h>
+#include <xparameters.h>
+#include "xil_printf.h"
+#include "spi_engine.h"
+#include "ad400x.h"
+#include "error.h"
+#include "delay.h"
+
+/******************************************************************************/
+/********************** Macros and Constants Definitions **********************/
+/******************************************************************************/
+#define AD400x_EVB_SAMPLE_NO			1000
```

Comment:
Maybe add these macro definitions in a separate file `parameters.h` as it is in the rest of the projects?